wd_demo.au3

```diff
@@ -86,32 +84,28 @@ Func _WD_Demo()
 			Case $idDebugging
 
 			Case $idButton
-				$bProcess = True
-				ExitLoop
+				RunDemo($idDebugging, $idBrowsers)
 
 			Case Else
 				For $i = 0 To $iCount - 1
 					If $aCheckboxes[$i] = $nMsg Then
 						$aDemoSuite[$i][1] = Not $aDemoSuite[$i][1]
-						ExitLoop
-
 					EndIf
-
 				Next
 		EndSwitch
 	WEnd
 
+	GUIDelete($hGUI)
+EndFunc   ;==>_WD_Demo
+
+Func RunDemo($idDebugging, $idBrowsers)
 	; Set debug level
 	$_WD_DEBUG = $aDebugLevel[_GUICtrlComboBox_GetCurSel($idDebugging)][1]
 
 	; Execute browser setup routine for user's browser selection
 	Local $sDesiredCapabilities = Call($aBrowsers[_GUICtrlComboBox_GetCurSel($idBrowsers)][1])
 
-	GUIDelete($hGUI)
-	If Not $bProcess Then Exit
-
 	_WD_Startup()
-
 	If @error <> $_WD_ERROR_Success Then
 		Exit -1
```

Comment:
This needs fixed as well since you no longer want the script to terminate. Should `Return` instead.